cmd/manager/main.go

```diff
@@ -225,6 +232,31 @@ func main() {
 		os.Exit(1)
 	}
 
+	err = mgr.Add(manager.RunnableFunc(func(ctx context.Context) error {
+		setupLog.Info(SUBOBJECT_DELETION_PROTECTION_MESSAGE)
+
+		akoPod := &corev1.Pod{}
+		err = mgr.GetClient().Get(ctx, client.ObjectKey{
```

Comment:
recheck if we need RBAC for this

Reply:
@igor-karpukhin @josvazg: we indeed do need additional RBAC here which we don't ship today.

Reply:
Removing emitting the event on the AKO pod for now. As we found, the inclusion of additional RBAC permissions is too intrusive for this release.